eng/pipelines/libraries/helix-queues-setup.yml

```
@@ -48,7 +48,7 @@ jobs:
 
     # Linux x64
     - ${{ if eq(parameters.platform, 'Linux_x64') }}:
-      - ${{ if eq(parameters.jobParameters.interpreter, '') }}:
+      - ${{ if and(eq(parameters.jobParameters.interpreter, ''), neq(parameters.jobParameters.isSingleFile, true)) }}:
```

```suggestion
      - ${{ if and(eq(parameters.jobParameters.interpreter, ''), not(eq(parameters.jobParameters.isSingleFile, true))) }}:
```

There is not `neq` in azure devops yaml schema 😢